civihr_employee_portal/src/Forms/OnboardingWebForm.php

```diff
@@ -234,11 +234,10 @@ private function getHelpText() {
    * @return string
    */
   private function getSkipButtonMarkup() {
-    $classes = 'btn btn-default-outline';
-    $style = 'float:none;';
+    $classes = 'btn btn-default chr_onboarding-wizard_remind-me-later';
     $format = '<button type="button" style="%s" class="%s">%s</button>';
     $buttonText = ts('Skip and Remind Me Later');
-    $buttonMarkup = sprintf($format, $style, $classes, $buttonText);
+    $buttonMarkup = sprintf($format, null, $classes, $buttonText);
```

Comment:
Instead of changing `$style` to NULL you can just remove it from the `$format` so it would become

```
     $format = '<button type="button" class="%s">%s</button>';
```

And then remove the argument completely from `sprintf`

```
$buttonMarkup = sprintf($format, $classes, $buttonText);
```